.eslintrc.json

```diff
@@ -1,15 +1,10 @@
 {
     "root": true,
     "parser": "@typescript-eslint/parser",
-    "extends": [
-        "eslint:recommended",
-        "plugin:@typescript-eslint/recommended",
-        "react-app",
-        "prettier"
-    ],
-    "plugins": ["@typescript-eslint", "prettier"],
+    "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+    "plugins": ["@typescript-eslint", "react", "react-hooks"],
```

Comment:
Is it advisable to include `react` and `react-hooks` here?

Reply:
Yes, without them you can't use ignore rules such as exhaustive dependency lists in `useEffect` hooks.

Reply:
Since the configuration is in the root, will including `react` have an effect on packages that don't use React?

Reply:
Alright.